inc/JACDAC/services/JDConsoleService.h

```diff
@@ -4,26 +4,26 @@
 #include "JACDAC.h"
 #include "ManagedString.h"
 
-#define JD_CONSOLE_LOG_PRIORITY_LOG         0
-#define JD_CONSOLE_LOG_PRIORITY_ERROR       1
-#define JD_CONSOLE_LOG_PRIORITY_WARNING     2
+#define JD_CONSOLE_LOG_PRIORITY_LOG         1
+#define JD_CONSOLE_LOG_PRIORITY_INFO        2
 #define JD_CONSOLE_LOG_PRIORITY_DEBUG       3
+#define JD_CONSOLE_LOG_PRIORITY_ERROR       4
```

Comment:
Are you sure about the order? debug < log < warn < error

Reply:
Is that the order? happy to modify